proto/v1/discovery.proto

```diff
@@ -45,6 +45,8 @@ message Details {
 
 // Provides information about the analysis status of a discovered resource.
 message Discovered {
+  reserved 2;
```

Comment:
You can just renumber the tag values since this is a brand new proto that nobody is using.

Reply:
Good point, done.